wgengine/tstun/tun.go

```diff
@@ -242,7 +263,10 @@ func (t *TUN) SetFilter(filt *filter.Filter) {
 // It blocks and does not take ownership of the packet.
 func (t *TUN) InjectInbound(packet []byte) error {
 	if len(packet) > MaxPacketSize {
-		return ErrPacketTooBig
+		return errPacketTooBig
+	}
+	if len(packet) == 0 {
+		return errPacketEmpty
```

Comment:
I'm not sure who calls InjectInbound and InjectOutbound right now, but would anything break if we just did `return nil` here and made (and documented) injecting an empty package a no-op?

Reply:
Nothing would break. Let me change that, then.